### PR TITLE
Change devtoolset-7 (scl-rh) of CentOS6

### PR DESCRIPTION
**Pull request by thinker0**
Usage http://mirror.centos.org/centos/6/sclo/x86_64/rh/devtoolset-7/

### CODE REVIEW

File: build-support/python/make-mesos-native-egg [hunk 1]
```diff
@@ -139,38 +139,33 @@ CENTOS6_X86_64_DEPENDENCIES=(
...
+  tar
 )
 build_centos6() {
   local mesos_version=$1 output_basedir=$2
   local python_outdir=$output_basedir/centos/6/python
   local image_name=centos:6
 
   pushd "$TMPDIR"
-    docker run -i -v $(pwd):/mesos-egg $image_name bash <<EOF
+    docker run -m=2g -i -v $(pwd):/mesos-egg $image_name bash <<EOF
```
Comment (rdelval) [inline build-support/python/make-mesos-native-egg]: Any particular reason CentOS6 requires the docker container to be limited to 2gb's of memory? Would be worth putting this as a comment if that's the case 🙂

File: build-support/python/make-mesos-native-egg [hunk 2]
```diff
@@ -139,38 +139,33 @@ CENTOS6_X86_64_DEPENDENCIES=(
...
     set -e -u
 
     # We need Software Collections for Python 2.7 (we dropped support for 2.6).
     # wget must get installed before everything else to fetch the additional repos.
-    yum -y install centos-release-SCL wget
-
-    # Mesos requires gcc 4.8, which is available in devtools-2 on centos 6.
```
Comment (rdelval) [inline build-support/python/make-mesos-native-egg]: Nit: We should leave a comment somewhere about why we're installing devtools-7 that way we know where to start looking if things break.

File: build-support/python/make-mesos-native-egg (hunk 1 above)
Comment (thinker0) [reply build-support/python/make-mesos-native-egg]: Ok, Removed

**Comment by thinker0** [comment]
Uploaded.
https://bintray.com/thinker0/mesos/mesos-executor-egg

**Pull request merged**